### PR TITLE
Add use case in Meitu Inc.

### PR DESCRIPTION
**Pull request by shafreeck**
# Contributing guidelines

Please read our [contribution workflow][contributing] before submitting a pull request.

[contributing]: https://github.com/coreos/etcd/blob/master/CONTRIBUTING.md#contribution-flow

### CODE REVIEW

File: Documentation/production-users.md [hunk 1]
```diff
@@ -118,3 +118,14 @@ PD(Placement Driver) is the central controller in the TiDB cluster. It saves the
 - *Backups*: None
 
 [nekto0n]:https://github.com/nekto0n
+
+## Meitu
+- *Application*: system configuration for services, service discovery, kubernetes in test environment
+- *Launched*: October 2015
+- *Cluster Size*: 1 clusters of 3 members
```
Comment (fanminshi) [inline Documentation/production-users.md]: `1 cluster of 3 members`?

**Comment by fanminshi** [comment]
@shafreeck change  the commit title to to `doc: Add use case in Meitu Inc.`

**Comment by fanminshi** [comment]
@shafreeck also resolve the merge conflict.

File: Documentation/production-users.md (hunk 1 above)
Comment (shafreeck) [inline Documentation/production-users.md]: Yes, sorry for typo

**Comment by shafreeck** [comment]
@fanminshi Thanks for your suggestions, the typo was fixed, and conflicts were resolved.  I rebased my branch according to upstream and made a force push to keep a clean commit history.

**Comment by xiang90** [comment]
LGTM. Thanks!

**Pull request merged**